OPTIONS.md

```diff
@@ -5,16 +5,16 @@ Alternative usage to run commands: webpack [command] [options]
 The build tool for modern web applications.
 
 Options:
-  -c, --config <value...>                                                            Provide path to a webpack configuration file e.g. ./webpack.config.js.
-  --config-name <value...>                                                           Name of the configuration to use.
+  -c, --config <pathToConfigFile...>                                                 Provide path to one or more webpack configuration files to process, e.g. "./webpack.config.js".
+  --config-name <name...>                                                            Name(s) of particular configuration(s) to use if configuration file exports an array of multiple configurations.
```

Comment:
It was generated by our script, so I think we need to improve this inside webpack-cli logic, otherwise it will be replaced in future

Reply:
I see, so there's nothing I can do from my side?

Reply:
I think you can.

Here we use `[value]`/`value` - https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/src/webpack-cli.ts#L678

But you can add a property (I think the best name is `valueDescription`) for https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/src/webpack-cli.ts#L805 (and for other) and use `const valueDescription = option.valueDescription ? valueDescription : "value"` (https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/src/webpack-cli.ts#L678
)

Reply:
I'm not sure I understand correctly.

From the code I have the impression that there is a [`description` property already](https://github.com/webpack/webpack-cli/blob/5134eebd9814036c0fac7c8a11f7a6cc211f64bc/packages/webpack-cli/src/webpack-cli.ts#L814).

So, I would need to add an additional property, (e.g. `parameterName`)? And you would amend your script to take this new property into account?

Reply:
No, when we register our options https://github.com/webpack/webpack-cli/blob/5134eebd9814036c0fac7c8a11f7a6cc211f64bc/packages/webpack-cli/src/webpack-cli.ts#L679 and https://github.com/webpack/webpack-cli/blob/5134eebd9814036c0fac7c8a11f7a6cc211f64bc/packages/webpack-cli/src/webpack-cli.ts#L681, that is why you see `value` instead `pathToConfigFile`/`name`/etc, we just need to add a new property to our options, for example https://github.com/webpack/webpack-cli/blob/5134eebd9814036c0fac7c8a11f7a6cc211f64bc/packages/webpack-cli/src/webpack-cli.ts#L952, i.e.
```
{
        name: "json",
        configs: [
          {
            type: "string",
          },
          {
            type: "enum",
            values: [true],
          },
        ],
        alias: "j",
        valueName: "pathToFile", // <- HERE
        description: "Prints result as JSON or store it in a file.",
        helpLev
``` 

And use `option.valueName` instead `value` here https://github.com/webpack/webpack-cli/blob/5134eebd9814036c0fac7c8a11f7a6cc211f64bc/packages/webpack-cli/src/webpack-cli.ts#L679 and https://github.com/webpack/webpack-cli/blob/5134eebd9814036c0fac7c8a11f7a6cc211f64bc/packages/webpack-cli/src/webpack-cli.ts#L681